### PR TITLE
2 papers till now ;)

### PR DESCRIPTION
**Pull request by ProtonStar12**

### CODE REVIEW

**Comment by GetPsyched** [comment]
Another name change will be required haha. It should be `eepc202-ms2...`, you've written `ms1` in the file name.

**Comment by GetPsyched** [comment]
Also, could you send the syllabi which has these subjects? I can't find them anywhere.

File: question-papers/eepc203-ms1-2023-sep.nix [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+      }]
+    ]
+    [
+      [{
+        text = ''
+          Calculate per phase per kilometer inductance of a bundled conductor
+          trasmission line given below.
+        '';
+        image = "https://i.imgur.com/8fOlHiW.jpeg";
+      }]
```
Comment (GetPsyched) [inline question-papers/eepc203-ms1-2023-sep.nix:43]: Marks are missing.

**Review by GetPsyched — CHANGES_REQUESTED** [review]

File: question-papers/eepc203-ms1-2023-sep.nix (hunk 1 above)
Comment (ProtonStar12) [reply question-papers/eepc203-ms1-2023-sep.nix]: done

**Review by GetPsyched — APPROVED** [review]
LGTM

**Pull request merged**